examples/integrations/test_postgres_integration.py

```diff
@@ -0,0 +1,95 @@
+# pip install psycopg2
+
+import uptrain
+import datetime
+import os
+
+
+os.environ['USERNAME'] = '...'
```

Comment:
add `POSTGRES_` prefix to these env variables?

Reply:
makes sense, added now